server/lib/payments.js

```diff
@@ -313,32 +306,32 @@ export const executeOrder = async (user, order, options) => {
     order.paymentMethod.save();
   }
 
+  // Credit card charges are synchronous. If the transaction is
+  // created here it means that the payment went through so it's
+  // safe to create subscription after this.
+
+  // The order will be updated to ACTIVE
+  order.interval && transaction && (await createSubscription(order));
+
```

Comment:
If for any reason the membership creation fails, we will still create the subscription. Shouldn't we move that after `Register user as collective backer` so that is the membership is not created we don't add the `subscription`?

Reply:
It was like that before. Why would the membership creation fail?

I thought the opposite, I wanted to make sure to create the subscription first and then take care of things like membership creation.

It's a bit out of scope and happy to change it back.

Reply:
> It was like that before.

From what I see it was at the end of the function before this PR no?

Reply:
Sorry, my answer was not clear at all. I definitely modified that and what you suggest is to go back to the previous behavior.

But, I'm not sure why we should create the subscription after the membership and not the opposite? It sounds more logic to create the subscription first, no?

Reply:
Thanks for clarifying! I would have put the membership creation before because the eventuality of having a member without subscription sounds better to me than the eventuality of having a subscription without a member. But it's not a strong argument, that's a preference. I was curious about the rational behind this change, I'm fine with it.